dashboard/app/views/projects_lists/index.html.haml

```diff
@@ -0,0 +1,13 @@
+- content_for :head do
+  %script{src:'/assets/js/projectsList.js'}
+
+#projects-list
+
+:ruby
+  reactProps = {
+    projectsData: @projects_list_data
+  }
+
+:javascript
+  ReactDOM.render(React.createElement(window.dashboard.ProjectsList, #{reactProps.to_json}),
+    document.getElementById('projects-list'));
```

Comment:
We've started taking a different approach to avoid using window.dashboard.ProjectList for things like this. Instead, we pass in the necessary data as a data-attribute on the script. As an example
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/sites/studio/pages/scriptOverview.js
https://github.com/code-dot-org/code-dot-org/blob/47ebd54cfa96ebdc9a04ba74fdc61f61bd2deea6/dashboard/app/views/scripts/show.html.haml#L76